algorithms/index.html

```diff
@@ -945,66 +860,344 @@
                 <li>If you have time, repeat for different ranges (0-500, 0-1000, etc).</li>
               </ul>
             </section>
+            <!-- END OF SEARCHING -->
 
+            <!-- START  OF SORTING ALGORITHMS -->
             <section>
-                <h1>Data Structures</h1>
-                <br>
-                <p>A way to store data (so that it can be used as desired)</p>
+                <h2>Sorting</h2>
             </section>
 
             <section>
-                <h2>Data Structures + Algorithms</h2>
-                <br>
-                <p>A good way to approach an algorithm is to think about the data structures that you know.</p>
+                <h2>Sorting</h2>
+                <span  class='left'>
+                <p>Why so important?</p>
                 <br>
-                <p>What are their properties, and how can they be used to your advantage?</p>
+                <p>You do it all the time in real life </p>
+                <ul>
+                    <li class='fragment'>The way your clothes are organized</li>
+                    <li class='fragment'>Where your dishes are in the kitchen</li>
+                    <li class='fragment'>Your books on your bookshelf</li>
+                </ul></br></br>
+                <p class='fragment'>They're not in perfect order, but in a way that makes it easier for you to search for things</p>
+            </span>
             </section>
 
             <section>
-                <h2>Example: Arrays</h2>
-                <img src="images/list.png">
+                <h3>Sorting</h3>
+                <span  class='left'>
+                <p>Many sorting algorithms for many types of data</p><br>
+                <ul>
+                <li class='fragment'>I sort my dishes by size and shape, not alphabetically</li>
+                <li class='fragment'>I sort my books alphabetically, not by color</li>
+                <li class='fragment'>When I sort my yarn I dump them all over the ground</li>
+                <li class='fragment'>When I sort my dishes I pull them out of the diswasher one at a time</li>
+                </ul></br></br>
```

Comment:
actually forget the notes comment from up above. maybe something about reverse chronological order here, like "I sort my receipts in reverse chronological order. Every time I get a new one, I put it on top of the pile"